signal_syscall.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2021-2022 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+//
+//go:build windows || aix || android || darwin || dragonfly || freebsd || hurd || illumos || ios || linux || netbsd || openbsd || solaris
```

Comment:
I don't think Windows should be on this list, it doesnt have SIGTERM afaik. Just os.Interrupt should suffice.

Reply:
`SIGTERM` is listed here: https://pkg.go.dev/syscall?GOOS=windows#pkg-constants

Reply:
Also, just tested on windows.

Reply:
I looked.  It was added in Go 1.14.  See the [Minor changes to the library -> os/signal in the release notes](https://go.dev/doc/go1.14).